projects/batfish-common-protocol/src/main/java/org/batfish/common/Container.java

```diff
@@ -34,35 +42,37 @@ public String getName() {
   }
 
   @JsonProperty(PROP_TESTRIGS)
-  public SortedSet<String> getTestrigs() {
+  public List<String> getTestrigs() {
     return _testrigs;
   }
 
-  @JsonProperty(PROP_ANALYSIS)
-  public Path getAnalysis() {
-    return _analysis;
-  }
-
-  @JsonProperty(PROP_NAME)
-  public void setName(String name) {
-    _name = name;
+  @JsonProperty(PROP_ANALYSES)
+  public Map<String, Analysis> getAnalyses() {
+    return _analyses;
   }
 
-  @JsonProperty(PROP_TESTRIGS)
-  public void setTestrigs(SortedSet<String> testrigs) {
-    _testrigs = testrigs;
+  public void addAnalysis(String analysisName, @Nullable Analysis analysis) {
```

Comment:
I think it would not be useful to pass a null analysis in this function, so "Nullable" does not seem necessary